docs/docs/manual/facets.md

```diff
@@ -0,0 +1,284 @@
+
+![A screenshot of an example numeric facet.](/img/numericfacet.png)
+
+Whereas a text facet groups unique text values into groups, a numeric facet sorts numbers by their range - smallest to biggest. This displays visually and allows you to set a custom facet within that range. You can drag the minimum and maximum range markers to set a range. OpenRefine snaps to some basic equal-sized divisions - 19 in the example set above. 
+
+You will be offered the option to include blank, non-numeric, and error values in your numeric visualization; these will appear in the visual range as “0” values.
+
+## Timeline facet
+
+![A screenshot of an example timeline facet.](/img/timelinefacet.png)
+
+Much like a numeric facet, a timeline facet will display as a small bar graph with the values sorted: in this case, chronologically. A timeline facet only works on dates formatted as “date” data types (e.g. by [using the `toDate()` function](expressions#dates) to transform text into dates, or by manually setting the [data type](#cell-data-types) on individual cells) and in the structure of the ISO-8601-compliant extended format with time in UTC: **YYYY**-**MM**-**DD**T**HH**:**MM**:**SS**Z. 
+
+## Scatterplot facet
+
+A scatterplot facet can be generated on any number-formatted column. You require two or more number columns to generate scatterplots. 
+
+You have the option to generate linear scatterplots (where the X and Y axes show continuous increases) or logarithmic scatterplots (where the X and Y axes show exponential or scaled increases). You can also rotate the plot by 45 degrees in either direction, and you can choose the size of the dot indicating a datapoint. You can make these choices in both the preview and in the facet display. 
+
+Going to “Facet” > “Scatterplot facet” will create a preview of data plotted from every number-formatted column in your dataset, comparing every column against every other column. Each scatterplot will show in its own square, allowing you to choose which data comparison you would like to analyze further. 
+
+When you click on your desired square, that two-column comparison will appear in the facets sidebar. From here, you can drag your mouse to draw a rectangle inside the scatterplot, which will narrow down to just the rows matching the points plotted inside that rectangle. This rectangle can be resized by dragging any of the four edges. To draw a new rectangle, simply click and drag your mouse again. To add more scatterplots to the facet sidebar, re-run this process and select a different square. 
+
+If you have multiple facets applied, plotted points in your scatterplot displays will be greyed out if they are not part of the current matching data subset. If the rectangle you have drawn within a scatterplot display only includes grey dots, you will see no matching rows.
+
+If you would like to export a scatterplot, OpenRefine will open a new tab with a generated PNG image that you can save. 
+
+## Custom text facet
```

Comment:
Is it worth mentioning somewhere that (unlike a Text facet) a Custom text facet does not support editing facet values to update the data in the cells used to create the facet?